charts/ceph-csi-cephfs/templates/provisioner-deployment.yaml

```diff
@@ -118,6 +118,7 @@ spec:
             - "--leader-election"
             - "--retry-interval-start=500ms"
             - "--handle-volume-inuse-error=false"
+            - "--feature-gates=AnnotateFsResize=true"
```

Comment:
why this is required for cephfs. we don't do explain on the node right?

Reply:
having an annotation populated in pv object does not matter either and good to be in parity between the drivers  in that sense, so added the same

Reply:
ptal thanks @Madhu-1